src/lib/journey/stages/webauthn.svelte

```diff
@@ -97,14 +98,16 @@
     form.submit();
   }
 
+  $: formMessageKey = convertStringToKey(form?.message);
```

Comment:
By separating this reactive block from the below reactive block, are we isolating `formMessageKey` changes from triggering a re-run of the reactive block below?